.github/CODEOWNERS

```diff
@@ -4,4 +4,4 @@
 # the repo. Unless a later match takes precedence,
 # they will be requested for
 # review when someone opens a pull request.
-*   @rosteen @javerbukh @pllim @kecnry @bmorris3 @haticekaratay @cshanahan1
+*   @rosteen @javerbukh @pllim @kecnry @bmorris3 @haticekaratay @cshanahan1 @gibsongreen
```

Comment:
I invited @gibsongreen to the Indigo Team. Once he accepted, this error would go away.

Reply:
Should be valid now, please confirm.

Reply:
I can see myself on the team page as of yesterday @pllim